content/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server.md

```diff
@@ -89,9 +89,9 @@ Maximum concurrency was measured using multiple repositories, job duration of ap
 {%- endif %}
 
 
-{%- ifversion ghes = 3.6 %}
+{%- ifversion ghes > 3.5 %}
 
-{% data reusables.actions.hardware-requirements-3.6 %}
+{% data reusables.actions.hardware-requirements-3.6-3.7 %}
```

Comment:
It looks like this is the only place we use this reusable. Instead of renaming, can we actually delete the reusable and put the content directly in the article? That way we won't have to worry about renaming in the future. 😊

Reply:
Should be a history of the previous GHES versions hardware requirements, there is currently the numbers from 3.2-3.7 ? Just changed the file to be 3.6-3.7, so as to not have to create a similar file, as the numbers didn't change

Reply:
Otherwise will proceed with removing the file 😊

Reply:
Instead of renaming the reusable file, we can delete it entirely and put its content directly into this article instead, just like the 3.2-3.5 recommendations above. Then we won't have to even think about renaming the file again if recommendations stay the same for 3.8.

Reply:
just updated!